spec/features/inputs/nested_select_input_spec.rb

```diff
@@ -165,26 +177,42 @@
       end
 
       it "sets display name on each select", js: true do
-        on_input_ctx("invoice_region_id") { expect_select2_selection(@metropolitana.id) }
-        on_input_ctx("invoice_city_id") { expect_select2_selection(@santiago.id) }
+        on_input_ctx("invoice_region_id") { expect_slimselect_selection(@metropolitana.id) }
+        on_input_ctx("invoice_city_id") { expect_slimselect_selection(@santiago.id) }
       end
 
       it "sets general minimum_input_length option", js: true do
-        msg = "Please enter 5 or more characters"
-        on_input_ctx("invoice_region_id") { open_select2_options }
-        expect_select2_result_text_to_eq(1, msg)
-        on_input_ctx("invoice_region_id") { select2_options_container.click }
-        on_input_ctx("invoice_city_id") { open_select2_options }
-        expect_select2_result_text_to_eq(1, msg)
+        msg = "Input is too short"
```

Comment:
⚠️ **[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Useless assignment to variable - msg.